File: backend/HolaSmileDMS/HDMS_API/Domain/Entities/Prescription.cs
public class Prescription
{
    [Key]
    public int PrescriptionId { get; set; }

    [ForeignKey("TreatmentRecord")]
    public int? TreatmentRecord_Id { get; set; }
    public TreatmentRecord? TreatmentRecord { get; set; }

    [ForeignKey("PrescriptionTemplate")]
    public int? Pre_TemplateID { get; set; }
    public PrescriptionTemplate? PrescriptionTemplate { get; set; }

    public string? Content { get; set; }

    public int? CreateBy { get; set; }
    public int? UpdatedBy { get; set; }

    public DateTime CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; }

    public bool IsDeleted { get; set; }
}

File: frontend/src/types/prescription.ts
// Types for prescription
export interface Prescription {
  prescriptionId: number;
  preTemplateId?: number;
  content: string;
  createdBy: string;
  updatedBy?: string;
  createdAt: string;
  updatedAt?: string;
  isDeleted: boolean;
  appointmentId: number;
}

export interface CreatePrescriptionRequest {
  appointmentId: number;
  contents: string;
}

export interface UpdatePrescriptionRequest {
  prescriptionId: number;
  contents: string;
}

export interface PrescriptionResponse {
  prescriptionId: number;
  appointmentId: number;
  content: string;
  createdAt: string;
  createdBy: string;
}